src/main/java/ca/concordia/soen6441/project/gameplay/PlayerImpl.java

```diff
@@ -37,6 +41,7 @@ public PlayerImpl(String p_name, ArrayList<String> p_ownedCountries, ArrayList<O
         this.d_Orders = p_Orders;
         this.d_Reinforcements = 0;
         this.d_HandsOfCardsManager = new HandOfCardsManager(this);
+        d_HandsOfCardsManager.addCard(new DiplomacyCard());

```

Comment:
You committed this by accident.  This needs to be removed ASAP.  Please be careful not to commit hardcoding for testing in the future.

Reply:
Fixed by myself.